source/ProSymbolEditor/Models/Symbol Attributes/DisplayAttributes.cs

```diff
@@ -0,0 +1,380 @@
+﻿/*******************************************************************************
+ * Copyright 2016 Esri
+ * 
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ * 
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ ******************************************************************************/
+
+using ArcGIS.Desktop.Framework.Contracts;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Web.Script.Serialization;
+
+namespace ProSymbolEditor
+{
+    public class DisplayAttributes : PropertyChangedBase
```

Comment:
Perhaps add a short summary on purpose of these classes: serialization container?